src/test/java/com/assignmentTest/Assignment6Test.java

```diff
@@ -0,0 +1,17 @@
+package com.assignmentTest;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import com.assignment.Assignment6;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Assignment6Test {
+
+    @Test
+    public void printCompanyReview() {
+    final Logger logger = LoggerFactory.getLogger(this.getClass());
```

Comment:
private Logger logger = LoggerFactory.getLogger(this.getClass()); , access modifer should be private, and it should be outside test at line 9